src/rules/orderedImportsRule.ts

```diff
@@ -109,6 +109,19 @@ function findUnsortedPair(xs: ts.Node[], transform: (x: string) => string): [ts.
     return null;
 }
 
+function sortByKey<T>(xs: T[], getSortKeyCallback: (x: T) => string): T[] {
+    return xs.sort((a, b) => {
+        const transformedA = getSortKeyCallback(a);
+        const transformedB = getSortKeyCallback(b);
+        if (transformedA > transformedB) {
+            return 1;
+        } else if (transformedA < transformedB) {
+            return -1;
+        }
+        return 0;
```

Comment:
In hindsight, beyond performance concerns, you might be better off doing things this way anyways. I believe `localeCompare` could sort strings differently depending on the users locale and then we wouldn't have a deterministic way of sorting things, which would be bad